site/content@beta/backend-apis/apis.md

```diff
@@ -0,0 +1,276 @@
+---
+weight: 40
+---
+
+# Agent Chat API
+
+## Use cases
+
+The **Agent Chat API** allows for: 
+
+- interaction with the chat as Agent (joining a chat, posting messages)
+- interact with the chat by Bot Agents (acting as Agents) 
+- building a custom Agent App
+- browsing chat archives
+- banning Customers 
+- and much more
+
+If you plan on using the **Agent Chat API** as **RTM API**, refer to [Agent Chat RTM API](../agent-chat-rtm-api). For the **Web API** usage, read the [Agent Chat Web API](../agent-chat-web-api) document.
+
+## Chatting as Agent
+
+The number of chats an Agent can participate in is not limited. It can be configured in the Agent App by the licence Owner or Admin.
+
+# Customer Chat API
+
+## Use cases
+
+The [**Customer Chat API**](../customer-chat-api) allows for:
+
+- interacting with the chat as Customer (join a chat, post messages)
+- building a custom Chat Widget
+- implementing new conversation channels (Facebook Messenger or Twitter)
+
+<!-- If you plan on using the **Customer Chat API** as **RTM API**, refer to [Customer Chat RTM API](../customer-chat-rtm-api). For the **Web API** usage, read the [Customer Chat Web API](../customer-chat-web-api) document. -->
+
+The **Customer Chat API** can also be used either as RTM API or Web API. For now, both usages are described in [one document]((../customer-chat-api)). We're currently working on this document, so expect changes soon!
+
+## Chatting as Customer
+
+By default, a Customer can only have one chat started, unless he chats with Agents who use different licences. The licence Owner or Admin can change that default value, modifying the `routing.max_customer_chats_count` property. However, in order to support this functionality, you would need to build a custom Chat Widget. For now, the LiveChat Chat Widget doesn't support it. You can build a Chat Widget from the ground up using the [Chat Widget Customer SDK](../customer-sdk/).
+
+Chats are continuous so Customers can always preview their chats' history. Yet, with **multiple concurrent chats** Customers can sort their chats out thematically, making sure they know what each chat is about. 
+
+# Configuration API
+
+## Use cases
+
+The [**Configuration API**](../configuration-api) allows for:
+
+- storing license configurations, 
+- creating chat, thread, and event [properties](#properties)
+- managing [webhooks](#webhooks), for example registering and unregistering
+- managing [Bot Agents](#bot-agents), for example creating and removing
+
+In the near future, it will allow for groups configuration. For now, refer to [**Platform REST API**](https://developers.livechatinc.com/docs/rest-api/) to manage groups properties.
+
+<!-- Czy ten przyklad wnosi jakas wartosc i w ogole jest dzialajacy? Pozbylabym sie go zupelnie i przyjela, ze w tym dokumencie nie ma kodu. Wtedy mozna sie pozbyc tego szarego bloku na kod po prawej i lepiej wyswietlac grafiki i animacje ktore dojda. -->
```

Comment:
Decided to comment out this example, because:
a) I'm not sure it's still correct and work properly
b) I don't know if it brings any value
c) it's a bit too long for a conceptual document